auth/console/policy_make_command.go

```diff
@@ -34,14 +34,42 @@ func (receiver *PolicyMakeCommand) Description() string {
 func (receiver *PolicyMakeCommand) Extend() command.Extend {
 	return command.Extend{
 		Category: "make",
+		Flags: []command.Flag{
+			&command.BoolFlag{
+				Name:    "force",
+				Aliases: []string{"f"},
+				Usage:   "Create the policy even if it already exists",
+			},
+		},
 	}
 }
 
 // Handle Execute the console command.
 func (receiver *PolicyMakeCommand) Handle(ctx console.Context) error {
 	name := ctx.Argument(0)
 	if name == "" {
-		return errors.New("Not enough arguments (missing: name) ")
+		var err error
+		name, err = ctx.Ask("Enter the policy name", console.AskOption{
+			Validate: func(s string) error {
+				if s == "" {
+					return errors.New("the policy name cannot be empty")
+				}
+
+				return nil
+			},
+		})
+		if err != nil {
+			return err
+		}
+	}
+
+	force := ctx.OptionBool("force")
+	path := receiver.getPath(name)
+	if !force {
+		if file.Exists(path) {
+			color.Red().Println("The policy already exists. Use the --force flag to overwrite")
+			return nil
+		}
```

Comment:
I think a lot of files are using the logic, can we create a public method for it?